templates/jenkins/files/README.md

```diff
@@ -1,19 +1,24 @@
-# Jenkins template for Platform.sh
+# Jenkins for Platform.sh
 
 This project provides a starter kit for Jenkins projects hosted on [Platform.sh](http://platform.sh).
 
-We include recommended .platform.app.yaml and .platform files that should suffice for most use cases. You are free to tweak them as needed for your particular site.
+Jenkins is an open source automation server written in Java. Jenkins helps to automate the non-human part of the software development process, with continuous integration and facilitating technical aspects of continuous delivery.
 
-## Platform.sh configuration files
+## Customizations
+
+The following files and additions make the Jenkins work.  If using this project as a reference for your own existing project, replicate the changes below to your project.
 
 * [`.platform/routes.yaml`](.platform/routes.yaml): Platform.sh allows you to define the [routes](https://docs.platform.sh/configuration/routes.html).
 * [`.platform/services.yaml`](.platform/services.yaml):  Platform.sh allows you to completely define and configure the topology and [services you want to use on your project](https://docs.platform.sh/configuration/services.html).
 * [`.platform.app.yaml`](.platform.app.yaml): You control your application and the way it will be built and deployed on Platform.sh [via a single configuration file](https://docs.platform.sh/configuration/app-containers.html).
 
-## References
+## Services
 
+* Java 8
+
+## References
```

Comment:
Do we want a References section?  I took that out of the few that had it in the other languages.  We should be consistent.

Reply:
I love to put references to a person to get further information.
What do you think?
If you think that is useful too, I can help to put it on the other templates.